CliClient/locales/zh_CN.po

```diff
@@ -1448,7 +1448,7 @@ msgstr "北极风格"
 
 #, fuzzy
 msgid "Show note counts"
-msgstr "显示托盘图标"
+msgstr "显示笔记数量"
```

Comment:
It is still marked as fuzzy, which means it will not show up in the translations.

Reply:
Hello. Did you see my comment?